modules/spring-session-ext/src/main/java/org/apache/ignite/spring/sessions/IgniteIndexedSessionRepository.java

```diff
@@ -384,38 +349,29 @@ private static TouchedExpiryPolicy createPolicy(Duration duration) {
      * A custom implementation of {@link Session} that uses a {@link MapSession} as the
      * basis for its mapping. It keeps track if changes have been made since last save.
      */
-    final class IgniteSession implements Session {
-
+    public class IgniteSession implements Session {
```

Comment:
Can we extract it from the parent class?

Reply:
Yep, you are right that using a nested class for a value looks bad? but let's do it in https://issues.apache.org/jira/browse/IGNITE-18302, because it requires making this class static or move it to upper level.